web/sessions.go

```diff
@@ -216,6 +236,34 @@ func (mss *mongoDBSessionStore) Get(ctx context.Context, id string) (*Session, e
 	return s, nil
 }
 
+func (mss *mongoDBSessionStore) GetByToken(ctx context.Context, token string) (*Session, error) {
```

Comment:
Discussed in person, consider switching to an exists query since we don't use the `Session` from this method in the added code